crates/store/re_tf/src/frame_id_registry.rs

```diff
@@ -1,27 +1,35 @@
 use std::collections::hash_map::Entry;
 
-use nohash_hasher::IntMap;
+use nohash_hasher::{IntMap, IntSet};
 use re_byte_size::SizeBytes;
-use re_log_types::EntityPath;
+use re_log_types::{EntityPath, EntityPathHash};
 use re_sdk_types::components::TransformFrameId;
 use re_sdk_types::{TransformFrameIdHash, archetypes};
 
-/// Frame id registry for resolving frame id hashes back to frame ids.
+/// Provides context around frame id hashes.
 pub struct FrameIdRegistry {
+    /// A lookup table for resolving frame id hashes back to frame ids.
     frame_id_lookup_table: IntMap<TransformFrameIdHash, TransformFrameId>,
+
+    /// Holds all frame ids per entity that were logged as a child frame.
+    child_frames_per_entity: IntMap<EntityPathHash, IntSet<TransformFrameIdHash>>,
```

Comment:
curious, should we for our purposes here also register parent frames? 🤔 
practically in a ROS scenario that probably means just including the root, so guess it doesn't matter! :)